build.gradle

```diff
@@ -83,6 +78,11 @@ def getDisplayVersion = { env, versionBase ->
     }
 }
 
+// Define configuration for config
```

Comment:
What is "configuration for config" supposed to tell me 🤔

Reply:
Amusingly this is actually correct, albeit certainly not very clear ... 😁 

It is a _Gradle_ configuration defined for the namespace of sorts used for the config _files_ - but probably a bit easier to reword this more to be about code analytics (even "codeMetrics" is somewhat odd). Made more sense when it was paired with other configuration namespaces in the engine project, I just took the "natives and" out without realizing how silly the comment ended up reading

Lots of it really is just old Gradle iterated on over and over then copy pasted elsewhere at odd times without really being streamlined much. In one of my pending Gradle overhauls some of these files have moved around more (so no more top level "config" directory, for instance)